boa/explorer.py

```diff
@@ -27,21 +47,22 @@ def fetch_abi_from_etherscan(
     address = _resolve_implementation_address(address, uri, api_key)
 
     # fetch ABI of `address`
-    params = dict(module="contract", action="getabi", address=address)
-    data = _fetch_etherscan(uri, api_key, **params)
+    data = _fetch_etherscan(
```

Comment:
is this change necessary?

Reply:
it was just weird to construct a dict to expand it immediately as kwargs